pipe-cli/src/utilities/storage/s3.py

```diff
@@ -225,17 +243,7 @@ def transfer(self, source_wrapper, destination_wrapper, path=None, relative_path
                 if not quiet:
                     click.echo('Skipping file %s since it exists in the destination %s' % (path, destination_key))
                 return
-        object_tags = ObjectTaggingManager.get_object_tagging(
-            ObjectTaggingManager(self.session, source_bucket, source_region), path)
-        if not tags and object_tags:
-            tags = self.convert_object_tags(object_tags)
-        if not self.has_required_tag(tags, 'CP_SOURCE'):
-            tags += ('CP_SOURCE=s3://{}/{}'.format(source_bucket, path),)
-        if not self.has_required_tag(tags, 'CP_OWNER'):
-            tags += ('CP_OWNER={}'.format(self._get_user()),)
-        TransferManager.ALLOWED_COPY_ARGS.append('Tagging')
         extra_args = {
-            'Tagging': self._convert_tags_to_url_string(tags),
```

Comment:
Both cli and api still pass tags whenever possible to cloud provider APIs because it provides possible data storage object tags backup source and it doesn't have any performance impact on the operations.